docs/_sources/user/subject_list_config.rst

```diff
@@ -279,3 +279,35 @@ The ``cpac_data_config_setup.py`` command line utility will produce a YAML file
         site: site01
 
 Note that more than one functional scan is defined under the `func` key (i.e., multiple series), and that individual scan parameters can be defined to override the default settings.
+
+Output Directory Ingress
+----------------
+
```

Comment:
The heading punctuation should be ["at least as long as the text"](https://www.sphinx-doc.org/en/master/usage/restructuredtext/basics.html#sections), and I think if we're saying "C-PAC now allows" we should drop in [a `versionadded` directive](https://www.sphinx-doc.org/en/master/usage/restructuredtext/directives.html#directive-versionadded) (or `versionchanged`?)

```suggestion
Output Directory Ingress
------------------------

.. versionadded:: 1.8.6

```

(If this "existed" pre-1.8.6, maybe something like
```RST
.. versionchanged:: 1.8.6
   «what changed»

.. versionadded:: 1.8.5
```